_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
 teaching: 0
 exercises: 0
 questions:
-- "What is user configuration file and how can I use it?"
+- "What is the user configuration file and how can I use it?"
 objectives:
-- "Understand the data directories structure"
+- "Understand the contents of the user-config.yml file"
+- "Prepare the user-config.yml file for use"
 - "Configure ESMValTool to ignore some settings"
```

Comment:
I am not sure I understand what you mean here. You want to set settings in the user-config.yml file so that the ESMValTool is ignoring some things it would normally do?